features/FxA-15-OAuth-Management/README.md

```diff
@@ -0,0 +1,83 @@
+Disconnect OAuth services from FxA
+======================
+
+https://mozilla.aha.io/features/FXA-15
```

Comment:
> as long as we have enough of a high-level idea of what the "Devices & apps" part will look like

Mockup above is very close to final: https://github.com/mozilla/fxa/pull/171#issuecomment-238696839